release-doc-in-process/all_vf.TTL

```diff
@@ -346,9 +366,87 @@ vf:fulfills     a owl:ObjectProperty ;
 #        rdfs:range      xsd:boolean ;
 #        rdfs:comment    "The commitment or process is complete or not." .
 
-# isSubstitutable "Defines if any resources of that type can be freely substituted for any other resource of that type when used, consumed, traded, etc."
+vf:isSubstitutable a    owl:DatatypeProperty ;
+        rdfs:label      "is substitutable" ;
+        rdfs:domain     vf:ResourceType ;
+        rdfs:range      xsd:boolean ;
+        rdfs:comment    "Defines if any resource of that type can be freely substituted for any other resource of that type when used, consumed, traded, etc." .
+
+#vf:unit a               owl:ObjectProperty ;
+#        rdfs:label      "unit" ;
+#        rdfs:domain     vf:ResourceType ;
+#        rdfs:range      qudt:unit ;
+#        rdfs:comment    "The unit expected for this resource type as a default." .
+
+vf:definedQuantity a    owl:ObjectProperty ;
+        rdfs:label      "defined quantity" ;
+        rdfs:domain     vf:RecipeFlow ;
+        rdfs:range      qudt:QuantityValue ;
+        rdfs:comment    "The amount and unit defined for this recipe flow." .
+
+# TODO: need to define this, something like currency, inventory, work, service; may be other aspects too.
+#vf:resourceCategory a    owl:ObjectProperty ;
+#        rdfs:label      "category" ;
+#        rdfs:domain     vf:ResourceType ;
+#        rdfs:comment    "Defines the category of behavior expected for the type of resource." .
+
+vf:developedBy a        owl:ObjectProperty ;
+        rdfs:label      "developed by" ;
+        rdfs:domain     vf:Recipe ;
+        rdfs:range      foaf:Agent ;
+        rdfs:comment    "The agent who developed this recipe." .
+
+vf:variantOf a           owl:ObjectProperty ;
+        rdfs:label      "variant of" ;
+        rdfs:domain     vf:Recipe ;
+        rdfs:range      vf:Recipe ;
+        rdfs:comment    "This recipe is a variation of another recipe." .
+
+vf:successorOf a        owl:ObjectProperty ;
+        rdfs:label      "successor of" ;
+        rdfs:domain     vf:Recipe ;
+        rdfs:range      vf:Recipe ;
+        rdfs:comment    "This recipe is an updated version of another recipe." .
+
+vf:affected a           owl:ObjectProperty ;
+        rdfs:label      "affected" ;
+        rdfs:domain     vf:RecipeFlow ;
+        rdfs:range      vf:RecipeResource ;
+        rdfs:comment    "Defines the type of resource this recipe flow affects." .
+
+vf:hasDuration a        owl:ObjectProperty ;
+        rdfs:label      "duration" ;
+        rdfs:domain     vf:RecipeProcess ;
+        rdfs:range      time:TemporalDuration ;
+        rdfs:comment    "The calendar duration defined for this type of process." .
+
+vf:durationMultiplier 
+        a               owl:DataTypeProperty ;
+        rdfs:label      "multiplier" ;
+        rdfs:domain     vf:RecipeProcess ;
+        rdfs:range      ???? ; #TODO
+        rdfs:comment    "The calendar duration defined for this type of process." .
+
+vf:resourceBasedOn
```

Comment:
I would not see a resource itself based on recipe. Just the flow that will *produce* that resource would stay *basedOn* a `FlowRecipe`. And again for now I would like to start with that `FlowRecipe` simply referencing a taxonomy with `vf:affectedClassifiedAs`. I don't think we need `RecipeResource` at all, or at least not to get started.

Reply:
To clarify, resource with same classification could get produced from various alternative recipes.

Reply:
>To clarify, resource with same classification could get produced from various alternative recipes.

Right, that is why the Recipe relates to the RecipeFlow.  And EconomicResource relates to resource type-thing. Yes, the resource type-thing could be the same but produced in different ways.  Maybe I misunderstand?

>I don't think we need RecipeResource at all, or at least not to get started.

How would you possibly use a recipe without it?  I see no need to put out a partial model that doesn't make sense, it will just confuse people.  Better to have none at all.  If we do a partial model, I would like it to make sense as it is.